rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -167,6 +167,12 @@
         }
       }
     },
+    "copyrightDate": {
+      "title": "Copyright Date",
+      "description": "Copyright Date of the resource.",
+      "type": "string",
```

Comment:
is it a date or a string? Can we put more constraints on this fields regex or other?

Reply:
regex pattern added